StudioCore/ParamEditor/ParamEditorScreen.cs

```diff
@@ -1049,35 +1062,63 @@ public void ParamView(bool doFocus)
                 }
                 scrollTo = 0;
 
-                List<int> pinnedRowList = new List<int>(_paramEditor._projectSettings.PinnedRows.GetValueOrDefault(activeParam, new List<int>()));
-                foreach (int rowID in pinnedRowList)
+                ImGui.Text("id VALUE | name ROW | prop FIELD VALUE | propref FIELD ROW\n | original | modified");
+                UIHints.AddImGuiHintButton("MassEditHint", ref UIHints.SearchBarHint);
+
+                //Goto ID
```

Comment:
space after "//"

Reply:
Gonna keep comment spacing in mind in the future, but for now how does a separate pull request that fixes the 1000+ current non-spaced comments sound?